azkaban_database_secrets.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_secretsmanager_secret" "azkaban_master_password" {
```

Comment:
Should these be in the repository in enterprise?

Reply:
Following the same pattern as done beforehand in ADG. https://github.com/dwp/aws-analytical-dataset-generation/blob/861735a7cb67b0ca720280b2c5abd1c0fe1e2795/metastore.tf#L169